projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -250,6 +245,14 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         return this._resourceStrings;
     }
 
+    /**
+     * Gets/Sets the display time for the row adding snackbar notification.
+     * @remarks
+     * By default it is 2000ms.
+     */
+    @Input()
+    public snackbarDisplayTime = 2000;
```

Comment:
There should definitely be an input for this, but the default is way too short. Until the user reads the info it's gone. Please bump the default up to 5000 or 7000 (test which is better).

Reply:
This default is based on @StefanIvanov 's decision. We can do it as long as you guys agree on.

Reply:
2 seconds is way too short to read and react. @StefanIvanov give your opinion please.

Reply:
I am fine with changing the default value to anything below 10000 so let's bump it up to 6000